src/client.cpp

```diff
@@ -1228,33 +1118,31 @@ std::string SdkVersion::toString() const
 }
 
 Status findCameraByName(Client& client, id_t accountId, const std::string& name,
-                        Instrument& cameraInfo)
+                        Feed& cameraInfo)
 {
-    Instruments instruments;
-    Status status = client.queryInstrumentsList(accountId, instruments);
+    Feeds feeds;
+    Status status = client.queryFeedsList(accountId, feeds);
```

Comment:
@andrii-prism The feed list can be very long. I think we should have `queryFeedsByName()` function in our API.

Reply:
@andrii-prism @dbus-prsm we have to fork connect ---> prismai_connect and virtual-camera ---> prismai_vc?